### DISCUSSION TITLE
More than 20x applying GitHub Student as I am an MSc Student at WorldQuant University, but my application never able to get processed

### DISCUSSION THREAD

**Discussion by alpha168hariyanto** [community/community#147860] [GitHub Education]
### Hi everyone,

Does anyone here know how to get approve for the submission Github education? I have tried more than 20x, but never get processed. 

Followings are the latest email from Github.

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Dec 24, 2024.

Common verification problems and suggested solutions are documented here: [Why wasn't my application approved?](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved).

Once you have resolved the issues with your sumbission, [visit us again and reapply](https://education.github.com/).

Have questions? Please seek help via our [Student Developer Pack Application FAQ ](https://github.com/orgs/community/discussions/111352)or visit the [GitHub Education Community.](https://github.com/orgs/community/discussions/categories/github-education)

💖 The GitHub Education Team

**Comment by AnglewoodJack** [comment]
I have the same problem and no luck in resolving whatsoever - there is simply no one to check student's affiliation other than computer algorithm, which keeps rejecting my transcript or any other proof of enrolment...
